core/src/main/java/hudson/model/Fingerprint.java

```diff
@@ -927,6 +923,18 @@ public Fingerprint(@CheckForNull Run build, @NonNull String fileName, @NonNull b
         return Util.toHexString(md5sum);
     }
 
+    /**
+     * Converts md5 hash string to byte array.
+     *
+     * @since TODO
+     */
+    public static byte[] toByteArray(String md5sum) {
```

Comment:
What does block you from using `string.getBytes()` with a 8-bit encoding? Why would it need a manual implementation?|